model-optimizer/extensions/back/PadNormalizer.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+from extensions.ops.ConvertLike import ConvertLike
+from mo.back.replacement import BackReplacementPattern
+from mo.front.tf.graph_utils import create_op_with_const_inputs
+from mo.graph.graph import Graph
+from mo.ops.const import Const
+
+
+class PadNormalizer(BackReplacementPattern):
+    enabled = True
```

Comment:
Please add description for this transformation.

Reply:
Absolutely, not clear why we need this transformation. When `Pad` operation is created in our graph, all data is in place to create this operation correctly without further need in normalization.